asserts/device_asserts.go

```diff
@@ -44,6 +44,16 @@ func (mod *Model) Model() string {
 	return mod.HeaderString("model")
 }
 
+// ModelDisplayName returns the display-friendly name of the model or
+// falls back to Model if this was not set.
+func (mod *Model) ModelDisplayName() string {
+	display := mod.HeaderString("model-display-name")
```

Comment:
@pedronis model-title is ok, certainly not too strange sounding. Personally, I think the "model-" prefix adds noise - we know it's the model's title because of the assertion type.